packages/safe-apps-provider/src/provider.ts

```diff
@@ -180,6 +180,9 @@ export class SafeAppProvider extends EventEmitter implements EIP1193Provider {
       case 'wallet_requestPermissions':
         return this.sdk.wallet.requestPermissions(params[0]);
 
+      case 'safe_settings':
```

Comment:
The method name is too vague IMO. You can't tell whether it is a read or write method just by looking at the name. Since it's a write method and you call the underlying SDK method call `setSafeSettings`, i believer the better name is `safe_setSettings`